contrib/epee/include/net/http_protocol_handler.inl

```diff
@@ -208,11 +208,46 @@ namespace net_utils
 		m_newlines(0),
 		m_bytes_read(0),
 		m_psnd_hndlr(psnd_hndlr),
-		m_conn_context(conn_context)
+		m_conn_context(conn_context),
+		m_initialized(false)
 	{
 
 	}
 	//--------------------------------------------------------------------------------------------
+	template<class t_connection_context>
+	simple_http_connection_handler<t_connection_context>::~simple_http_connection_handler()
+	{
+	  try
+	  {
+	    if (m_initialized)
+	    {
+	      CRITICAL_REGION_LOCAL(m_config.m_lock);
+        if (m_config.m_connection_count)
```

Comment:
Minor style issue (not material for approval): If GitHub browser view is correct, the indent here appears to differ from elsewhere.

Reply:
It's a tab vs space issue. This file has some funky indenting. Is it worth force pushing an update over?

Reply:
Yea I figured, that's why I added this comment post-approval, to clarify that I don't care much. I would say just do whatever you want here, if you're pushing a change anyways to warn of inconsistent configs, then maybe it's trivial to include.

Reply:
Fixed.